sdk/formrecognizer/azure-ai-formrecognizer/src/main/java/com/azure/ai/formrecognizer/training/CustomModelTransforms.java

```diff
@@ -110,7 +111,13 @@ static CustomFormModel toCustomFormModel(Model modelResponse) {
                 PrivateFieldAccessHelper.set(customFormModel, "trainingDocuments",
                     trainingDocumentInfoList);
             }
+        } else {
+            // default to false
+            PrivateFieldAccessHelper.set(customFormModelProperties, "isComposed", false);
```

Comment:
python/.net are using `isComposedModel`

Reply:
The default value of java's primitive type, boolean, is false already.

Reply:
But we weren't setting `customModelProperties` so doing that too :)